Diagnostics/HealthChecker/Tests/HealthChecker.Tests.ps1

```diff
@@ -239,6 +240,7 @@ Describe "Testing Health Checker by Mock Data Imports" {
             Assert-MockCalled Get-OrganizationConfig -Exactly 1
             Assert-MockCalled Get-HybridConfiguration -Exactly 1
             Assert-MockCalled Get-Service -Exactly 1
+            Assert-MockCalled Get-SettingOverride -Exactly 1
             Assert-MockCalled Get-ServerComponentState -Exactly 1
             Assert-MockCalled Test-ServiceHealth -Exactly 1
         }
```

Comment:
Also add actual testing results besides that we just called it. Maybe add it to "Checking Scenarios 1"

Reply:
Added `AMSI Override` and `AMSI Enabled` test to `Checking Scenarios 1`. Please have a look.